File: Cursojava22/app/src/main/java/com/example/cursojava2/FragmentActivity.java
package com.example.cursojava2;

import android.net.Uri;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class FragmentActivity extends AppCompatActivity implements FregUno.OnFragmentInteractionListener, FregDos.OnFragmentInteractionListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_fragment);
    }

    @Override
    public void onFragmentInteraction(Uri uri) {

    }

}

File: KotlinCurso2/app/src/main/java/com/example/kotlincurso2/FragmentActivity.kt
package com.example.kotlincurso2

import android.net.Uri
import android.support.v7.app.AppCompatActivity
import android.os.Bundle

class FragmentActivity : AppCompatActivity(),FregUno.OnFragmentInteractionListener, FregDos.OnFragmentInteractionListener  {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_fragment)


    }

    override fun onFragmentInteraction(uri: Uri) {

    }

}
